src/include/functions_frontendhelpers.php

```diff
@@ -122,7 +122,7 @@ function GetAdditionalUrl($skipParam, $appendParam = "")
 function CreateCurrentUrl()
 {
 	global $content;
-	$content['CURRENTURL'] = $_SERVER['PHP_SELF']; // . "?" . $_SERVER['QUERY_STRING']
+	$content['CURRENTURL'] = ltrim($_SERVER['PHP_SELF'],'/'); // . "?" . $_SERVER['QUERY_STRING']
```

Comment:
That breaks the currenturl on other plattforms!
Can you check what the output of $_SERVER['PHP_SELF'] is at your end?

Reply:
$_SERVER['PHP_SELF'] in a separate "selftest.php" returns /selftest.php. This also happens when being called via the reverse proxy.

Reply:
Ok I undersrand. But that patch only solves the issue for you when installaing loganalyzer into the root directory. 

I will look into this and find a fix that works when loganalyzer is running in root or relative pathes.

Reply:
Please tell me your PHP and Linux Version.
Something is different to the version I have been testing with. 

Can you send me output of phpInfo(); of your machine?

Reply:
And something else you can test in your enviromemtn: 

instead of $_SERVER['PHP_SELF']
use $_SERVER['REQUEST_URI']

Please let me know if that generates correct links at your end?